### DISCUSSION TITLE
any plans for 32-bit?

### DISCUSSION THREAD

**Discussion by ghost** [SpiralLinux/SpiralLinux-project#59] [Q&A] [answered]
I was curious if this projects has plans to include support for 32-bit systems down the line seeing as Debian still does. It can be a pain getting Debian user-friendly on an older PC.

**Comment by geckolinux** [comment, marked as answer]
Hi there, thanks for the question. I haven't discarded the possibility, but no immediate plans. I'm not sure how much additional work it would take to modify and build the image recipes for 32-bit. But I also have an old 32-bit laptop that currently runs vanilla Debian, and it was indeed a major pain to install and configure.
